src/services/mongoDomainService.ts

```diff
@@ -7,16 +8,18 @@ import { DomainService } from "./domainService";
 //Remake to MONGODomainService implemented with mongo
 export class MongoDomainService extends DomainService<MongoDbService> {
   dbClient: MongoClient;
-
-  constructor(client: MongoClient, service: MongoDbService) {
+  context: Context;
```

Comment:
Is there a way to eliminate this code smell of having Azure Functions `Context` be inside of the MongoDomainService?

Could we instead just have an interface we pass which has the functions we care about (like `log`)

Reply:
I can make use of the Azure Functions Logger interface, the package provides one.